ThymeAppDelegate.m

```diff
@@ -214,17 +214,24 @@ - (IBAction)onAboutClick:(id)sender {
 }
 
 - (void)updateStatusBar {
+    NSStatusBarButton *button = [statusItem button];
+    
     if ([self.stopwatch isStopped]) {
-        [statusItem setLength:26.0];
```

Comment:
The docs said `setTitle` etc. were deprecated and you should now use the button. If this is breaking anything, it can go, of course.